packages/dx-vue-grid/docs/reference/table-band-header.md

```diff
@@ -0,0 +1,80 @@
+# DxTableBandHeader Plugin Reference
+
+A plugin that renders the banded cells.
+
+## Importing
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { DxTableBandHeader } from '@devexpress/dx-vue-grid-bootstrap4';
+```
+
+If you want to use custom components, you can import the themeless plugin:
+
+```js
+import { DxTableBandHeader } from '@devexpress/dx-vue-grid';
+```
+
+## User Reference
+
+### Dependencies
+
+- [DxTable](table.md)
+- [DxTableHeaderRow](table-header-row.md)
+- [DxTableEditColumn](table-edit-column.md) [Optional]
+- [DxTableSelection](table-selection.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+cellComponent | ComponentType&lt;[DxTableBandHeader.DxCell](#dxtablebandheaderdxcell)&gt; | | A component that renders a band cell.
```

Comment:
We don't use the `ComponentType<...>` for Vue Grid. Let's keep `DxTableBandHeader.DxCell`.